app/views/root/_base.html.erb

```diff
@@ -24,6 +24,31 @@
       } %>
     </div>
   </form>
+
+  <div id="accounts-signed-out" class="govuk-header__content gem-c-header__content" data-module="govuk-header">
+    <button role="button" class="govuk-header__menu-button gem-c-header__menu-button govuk-js-header-toggle" aria-controls="navigation" aria-label="Show or hide Top Level Navigation">Menu</button>
+    <nav class="gem-c-header__nav">
+      <ul id="navigation" class="govuk-header__navigation govuk-header__navigation--end" aria-label="Top Level Navigation">
+        <li class="govuk-header__navigation-item">
+          <a class="govuk-header__link" href="https://www.gov.uk/transition-check/login">Sign in</a>
+        </li>
+      </ul>
+    </nav>
+  </div>
+
+  <div id="accounts-signed-in" class="govuk-header__content gem-c-header__content" data-module="govuk-header">
+    <button role="button" class="govuk-header__menu-button gem-c-header__menu-button govuk-js-header-toggle" aria-controls="navigation" aria-label="Show or hide Top Level Navigation">Menu</button>
+    <nav class="gem-c-header__nav">
+      <ul id="navigation" class="govuk-header__navigation govuk-header__navigation--end" aria-label="Top Level Navigation">
+        <li class="govuk-header__navigation-item">
+          <a class="govuk-header__link" href="https://www.account.publishing.service.gov.uk/">Account</a>
+        </li>
+        <li class="govuk-header__navigation-item">
+          <a class="govuk-header__link" href="https://www.gov.uk/transition-check/logout">Sign out</a>
```

Comment:
I've just had a thought; can we make these links different on staging?  www.gov.uk work for that

Reply:
On staging we'd ideally have www.staging.publishing.service.gov.uk instead of www.gov.uk links and www.account.staging.publishing.service.gov.uk instead of www.account.publishing.service.gov.uk